docs/xxcriurestorenonportablemode.md

```diff
@@ -0,0 +1,61 @@
+<!--
+* Copyright (c) 2017, 2023 IBM Corp. and others
+*
+* This program and the accompanying materials are made
+* available under the terms of the Eclipse Public License 2.0
+* which accompanies this distribution and is available at
+* https://www.eclipse.org/legal/epl-2.0/ or the Apache
+* License, Version 2.0 which accompanies this distribution and
+* is available at https://www.apache.org/licenses/LICENSE-2.0.
+*
+* This Source Code may also be made available under the
+* following Secondary Licenses when the conditions for such
+* availability set forth in the Eclipse Public License, v. 2.0
+* are satisfied: GNU General Public License, version 2 with
+* the GNU Classpath Exception [1] and GNU General Public
+* License, version 2 with the OpenJDK Assembly Exception [2].
+*
+* [1] https://www.gnu.org/software/classpath/license.html
+* [2] https://openjdk.org/legal/assembly-exception.html
+*
+* SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH
+* Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
+-->
+
+# -XX:\[+|-\]CRIURestoreNonPortableMode
+
+**(Linux&reg; x86, Linux on POWER LE&reg;, and Linux on IBM Z&reg; only)**
+
+This option enables and disables the JIT compiler from generating nonportable compiled code on restore.
+
+:fontawesome-solid-triangle-exclamation:{: .warn aria-hidden="true"} **Restrictions:**
+
+- This option takes effect only when [`-XX:+EnableCRIUSupport`](xxenablecriusupport.md) is enabled.
+- When this option is disabled, CRIUSECProvider is the only security provider in both checkpoint and restore.
```

Comment:
What is CRIUSECProvider? There is no mention of this anywhere in the existing docs. If there is a new security provider, surely it should be documented somewhere?

Reply:
Its described in the javadocs, https://eclipse-openj9.github.io/openj9-docs/api-jdk11/ look for [org.eclipse.openj9.criu.SecurityProvider](https://eclipse-openj9.github.io/openj9-docs/api/jdk11/openj9.criu/org/eclipse/openj9/criu/package-summary.html)